test/integration/graphql/schema.test.js

```diff
@@ -195,6 +195,13 @@ describe('schema', () => {
       findBreakingChanges(
         schema,
         buildSchema(`#graphql
+          directive @auth on OBJECT | FIELD_DEFINITION
+
+          """
+          Indicates exactly one field must be supplied and this field must not be null.
+          """
+          directive @oneOf on INPUT_OBJECT
+
```

Comment:
whats this for?

Reply:
Not sure. Must have come from when I copied the schema. I've removed it.